### PR TITLE
Update CHANGELOG.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

**Comment by brendandburns** [comment]
Closes #6282

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,117 @@
...
+   * Fix apiserver proxy path rewriting
+   * Upgrade to etcd 2.0.0
+   * Add a wordpress/mysql example
+   * Improve responsiveness of the master when creating new pods
+   * Improve api object validation in numerous small ways
+   * Add support for IPC namespaces
+   * Improve GCE PD support
+   * Make replica controllers with node selectors work correctly
+   * Lots of improvements to e2e tests (more to come)
+
```
Comment (fabioy) [inline CHANGELOG.md]: Could you reformat this a bit? Previous entries used a "-" and had consistent left spacing for each entry.

Thanks.

Comment (brendandburns) [inline CHANGELOG.md]: The markdown renders 100% exactly the same.  Do you really want me to do this?  (I copy/pasted from the release notes page)

**Comment by fabioy** [comment]
Eh, as long as the markdown looks ok. LGTM.

**Pull request merged**